frontend/src/components/Review/Review.tsx

```diff
@@ -162,6 +180,15 @@ const ReviewComponent = ({
                       <ExpandMoreIcon />
                     </IconButton>
                   </Grid>
+
+                  {/* Checking to see if aptId of the review is null or empty, indicating that the review is a 
+                  landlord apartment - if landlord apartment is true, will add landlord review text */}
+                  <Typography className={landlordIndicator}>
+                    {review.aptId === '' || review.aptId == null
```

Comment:
I think you can do `!review.aptId` instead.

Also, with what I was saying above, if you add `landlordName` as a prop, you can rewrite the code as such:

```
{landlordName && (
<Typography className={landlordIndicator}>
  'Landlord Review - ' + landlordName
</Typography>
)}
```

This also only renders the component if `landlordName` has an input.